tests/integration/test_ses.py

```diff
@@ -109,9 +110,23 @@ def test_send_email_can_retrospect(self, ses_client):
         emails_url = config.get_edge_url() + INTERNAL_RESOURCE_PATH + EMAILS_ENDPOINT
         api_contents = requests.get(emails_url).json()
         api_contents = {msg["Id"]: msg for msg in api_contents["messages"]}
+        assert len(api_contents) == 1
```

Comment:
```suggestion
        assert len(api_contents) >= 1
```
I'd loosen the assertion here a bit, to make the tests more resilient against side effects (e.g., if other tests are also publishing messages that are getting tracked in the `EMAILS` dict).